package.json

```diff
@@ -87,7 +88,7 @@
     "@babel/preset-env": "^7.20.0",
     "@babel/runtime": "^7.20.0",
     "@formatjs/cli": "^6.2.0",
-    "@mapeo/core": "^9.0.0-alpha.5",
+    "@mapeo/core": "file:mapeo-core-9.0.0-alpha.5.tgz",
```

Comment:
As you might imagine, I intend to replace these with real versions soon. These are built from https://github.com/digidem/mapeo-core-next/pull/550.